lib/sprockets/media_query_combiner/processor.rb

```diff
@@ -1,15 +1,40 @@
-require 'tilt'
 require 'sass/media_query_combiner/combiner'
 
 module Sprockets
   module MediaQueryCombiner
-    class Processor < Tilt::Template
-      def prepare
+    
+    # serves to pick proper processor
+    class Processor
+      # returns processor class valid for current Sprockets version
+      def self.choose
+        return SprocketsProcessor if Sprockets.respond_to? :register_postprocessor
```

Comment:
Is this the only way to feature detect this? It seems totally unrelated so is no better than a version check. If there's not a better way, I'd rather just use a version check.